### PR TITLE
Bwtester path selection and panic recovery

### PR DESCRIPTION
**Pull request by FR4NK-W**
Added standard scionproto logging
Recover from panic and log stack trace
Added path selection in bwtesterclient when in interactive mode

### CODE REVIEW

File: bwtester/bwtestlib/bwtestlib.go [hunk 1]
```diff
@@ -250,3 +270,39 @@ func HandleDCConnReceive(bwp *BwtestParameters, udpConnection *snet.Conn, res *B
 	}
 	_ = udpConnection.Close()
 }
+
+func ChoosePath(interactive bool, local snet.Addr, remote snet.Addr) *sciond.PathReplyEntry {
+	pathMgr := snet.DefNetwork.PathResolver()
+	pathSet := pathMgr.Query(local.IA, remote.IA)
+	pathIndices := make(map[uint64]pathmgr.PathKey)
```
Comment (perrig) [inline bwtester/bwtestlib/bwtestlib.go]: I would suggest to create a slice instead of a map

File: bwtester/bwtestlib/bwtestlib.go [hunk 2]
```diff
@@ -250,3 +270,39 @@ func HandleDCConnReceive(bwp *BwtestParameters, udpConnection *snet.Conn, res *B
...
+		for {
+			fmt.Printf("Choose path: ")
+			pathIndexStr, _ := reader.ReadString('\n')
+			pathIndex, err := strconv.ParseUint(pathIndexStr[:len(pathIndexStr)-1], 10, 64)
+			if err == nil && pathIndex < uint64(len(pathIndices)) {
+				break
+			}
+			fmt.Printf("ERROR: Invalid path index, valid indices range: [0, %v]\n", i-1)
+		}
+	}
```
Comment (perrig) [inline bwtester/bwtestlib/bwtestlib.go]: else pick shortest path. Todo: write path quality evaluator.

File: vendor/vendor.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{
```
Comment (perrig) [inline vendor/vendor.json]: Please update install instructions, to reflect the use of govendor

File: bwtester/bwtestlib/bwtestlib.go (hunk 1 above)
Comment (FR4NK-W) [reply bwtester/bwtestlib/bwtestlib.go]: Done, also improved user input handling

File: bwtester/bwtestlib/bwtestlib.go (hunk 2 above)
Comment (FR4NK-W) [reply bwtester/bwtestlib/bwtestlib.go]: Done
Added basic path selection function

File: vendor/vendor.json (hunk 3 above)
Comment (FR4NK-W) [reply vendor/vendor.json]: Done

**Pull request merged**